nest/00. Foreword.md

```diff
@@ -0,0 +1,24 @@
+# Getting started with Nest
+
+## Foreword
+
+Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
+
+In order to avoid any confusion, we will refer to the project as a whole as `Nest` (with an upper case n), while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.
+
+[//]: # (TODO: add link to the installation instructions page of Raven-OS)
+Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
+
+## How to read this guide
+
+nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. That latter is targeted towards more advanced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
+
+[//]: # (TODO: Add link to the chapter 1)
+[//]: # (TODO: Add link to the chapter 2)
+[//]: # (TODO: Add link to the chapter 3)
+[//]: # (TODO: Add link to the chapter 4)
+The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentionned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
+
+## Getting help
+
+If you need help, you can join our official [communautary Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.
```

Comment:
`Communautary` doesn't seem to be a real word, what you're looking for is probably `Communal`, but I think `Community` fits best here.
```suggestion
If you need help, you can join our official [community Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.
```